### ISSUE TITLE
Workflow bookmark filter

### ISSUE THREAD

**Issue by RZ9082** [galaxyproject/galaxy#18718]
In the workflow tab, when '★ Show bookmarked' is selected, only bookmarked workflows on the first page are displayed!

**Comment by wm75** [comment]
Yes, seems the query issued when clicking the filter is a standard /api/workflows?sort_by=update_time&sort_desc=true&limit=24&offset=0&search=&show_published=false&skip_step_counts=true one when the search should be over all your WFs

**Comment by itisAliRH** [comment]
You're right! The issue is that the bookmarked workflows are filtered locally, and the API doesn't support this yet. I'll add this to the API and fix it.
